OpenSim/Common/Component.h

```diff
@@ -2395,94 +2384,98 @@ template<class T>
 void Input<T>::connect(const AbstractOutput& output,
                        const std::string& annotation) {
     const auto* outT = dynamic_cast<const Output<T>*>(&output);
```

Comment:
The rest of the changes in this file are simply rearranging code and renaming variables. The diff really makes this section hard to read, but I really didn't make substantial changes here.